<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="JunctionNumInput">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="JunctionNunLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Junction number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IterationsNumInput">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="IterationsNumLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Number of iterations</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DisplayButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DisplayTypeComboBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>115</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Absolute voltage</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cumulative voltage</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="DisplayTypeLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Display type</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
